marlowe-runtime/cip30-demo/Main.hs

```diff
@@ -21,12 +38,25 @@ newtype WalletName = WalletName String
     , Blaze.ToMarkup
     )
 
+data PostWalletCreateRequestDTO = PostWalletCreateRequestDTO
+  { version :: Core.Api.SomeMarloweVersion
+  , source :: Aeson.Value
```

Comment:
Change this to `contract :: Contract` and instead of setting this to the text value of the form, call `JSON.parse` and assign the resulting object tot it.